.travis.yml

```diff
@@ -25,6 +25,7 @@ env:
     - JAVA_OPTS="-XX:+TieredCompilation -XX:TieredStopAtLevel=1"
     - MALLOC_ARENA_MAX=2
   matrix:
+    - JTR='ci --batch ../lib/ruby/truffle/jruby+truffle/gem_ci/travis.txt'
```

Comment:
It's also at the end of the matrix. It seems this first PR commit overlaps with #3671.